packages/dm-core-plugins/src/job/JobPlugin.tsx

```diff
@@ -79,12 +83,7 @@ export const JobPlugin = (props: IUIPlugin & { config: JobPluginConfig }) => {
   const [jobExists, setJobExists] = useState(false)
   const [result, setResult] = useState<GetJobResultResponse | null>(null)
   const [asCronJob, setAsCronJob] = useState<boolean>(false)
-  const [jobSchedule, setJobSchedule] = useState<TSchedule>({
-    type: EBlueprint.CRON_JOB,
-    cron: defaultCron,
-    startDate: '',
-    endDate: '',
-  })
+  const [jobSchedule, setJobSchedule] = useState<TSchedule>(emptyJob)
   const canSubmit =
     !asCronJob || Boolean(jobSchedule.startDate && jobSchedule.endDate)
```

Comment:
Know this is outside current PR.
But, no need to set this requirement on start- endDate when we don't use them for anything

Reply:
We can disable the check, but I suggest that we do it in another PR